packages/ra-ui-materialui/src/input/ArrayInput.tsx

```diff
@@ -85,6 +109,15 @@ export const ArrayInput = ({
                     isRequired={isRequired(validate)}
```

Comment:
`isRequired` is already a boolean

Reply:
Ignore previous comment. As you'll not be using `useInput`, you'll have to call `isRequired(validate)`. Just not the `isRequired` from the `useInput` but the function imported